### ISSUE TITLE
2019-11-20 tc010 M1SD test case variant with all Concentration values set to NA. This causes computation engine to crash. Expectation is that it would not crash but set requested parameters to NA

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#36]
This WAS working appropriately in current master branch.

```
d$PKCNCN <- rep(NA, nrow(d)) # computation engine should not FALL OVER

> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : Error in tmp[[2]]: subscript out of bounds
For SDEID 2925168
In addition: Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : Error in tmp[[2]]: subscript out of bounds
For SDEID 2925168 

```

**Comment by jhhughes256** [comment]
### Update

This specific error occurs for any test case with partial AUC. Notably, both tc014 and tc016 do not experience this error and do not include partial AUCs in the base mct. When adding partial AUCs to any testcase other than tc013 this error can be replicated.

tc012 and tc013 do not include partial AUCs in the base mct but crash without the addition of partial AUCs. Additional information on these specific errors can be found in issue #27 .

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
# Still occurs with tc010

## read data
``` 
testcase.id <- "tc010"
p <- "./testcases/tc010"
params["INPUT1"] = file.path(p, "B1261002.in.csv")
params["parameterset"] ="PARAMETERLIST"
params["map"] = file.path(p, "tc010_mct_updated.csv")
params["flags"] = file.path(p, "B1261002_flag_data.in.csv")
params["OUTPUT"] ="."
print(params)
  
mct <- read_csv(params["map"])
write_file(toJSON(mct, na="string"), path=file.path(p, "tc010_mct_updated.json"))
params["map"] = file.path(p, "tc010_mct_updated.json")
  
flags <- read_csv(params["flags"])
write_file(toJSON(flags, na="string"), path=file.path(p, "tc010_flags.json"))
params["flags"] = file.path(p, "tc010_flags.json")
```

## prepare input concentration dataframe, mct, flags

```
conc <- read.csv(file=params["INPUT1"], header=TRUE, sep=",", as.is=TRUE)
vlist <- c("PKDATAROWID", "PKPTMS", "PKCNCN", "PERIOD", "TREATXT")
  
# Read mct
mct <- params["map"]
###mct <- read_file(mct)
###mct <- gsub("[\\]+", "", mct)
mct <- fromJSON(mct)
  
# Read flags
flags <- params["flags"]
###flags <- read_file(flags)
###flags <- gsub("[\\]+", "", flags)
flags <- fromJSON(flags)
  
# ParameterSet
if(is.na(params["parameterset"])) { params["parameterset"] <- "PARAMETERDISPLAYLIST" }
parameterset <- params["parameterset"]
  
# FLAGS is a subset for the specific profile, subset concentration dataframe 
#  to those records by PKDATAROWID
d <- conc[is.element(conc$PKDATAROWID, flags$PKDATAROWID),]
```

## execute test case

```
> model_spec(mct)
[1] "openNCA: Model: M1 DosingType: SD AUCMETHOD: LINLOG"
> d$PKCNCN <- rep(NA, nrow(d)) # computation engine should not FALL OVER
> parameterset
   parameterset 
"PARAMETERLIST" 
> 
> # Parmeters for M1SD
> model_parameters("M1|M1SD")
  [1] "AUCALL"       "AUCALLDN"     "AUCDN"        "AUCINFO"      "AUCINFOC"     "AUCINFODN"    "AUCINFOi"     "AUCINFP"     
  [9] "AUCINFPC"     "AUCINFPDN"    "AUCINFPi"     "AUCLAST"      "AUCLASTi"     "AUCLASTC"     "AUCLASTCi"    "AUCLASTDN"   
 [17] "AUCLASTDNi"   "AUCT"         "AUCT1_T2"     "AUCTAUDNi"    "AUCTAU"       "AUCTAUi"      "AUCTDN"       "AUCXPCTO"    
 [25] "AUCXPCTOi"    "AUCXPCTP"     "AUCXPCTPi"    "AUMCINFO"     "AUMCINFOi"    "AUMCINFP"     "AUMCINFPi"    "AUMCLAST"    
 [33] "AUMCLASTi"    "AUMCTAUi"     "AUMCXPTO"     "AUMCXPTOi"    "AUMCXPTP"     "AUMCXPTPi"    "CAVi"         "CEST"        
 [41] "CLAST"        "CLASTi"       "CLFO"         "CLFOW"        "CLFP"         "CLFPW"        "CLFTAUi"      "CLFTAUWi"    
 [49] "CMAX"         "CMAXi"        "CMAXDN"       "CMAXDNi"      "CMIN"         "CMINi"        "CMINDN"       "CMINDNi"     
 [57] "CTROUGHi"     "CTROUGHENDi"  "DIi"          "DOSE"         "DOSEi"        "DOSEC"        "DOSECi"       "F"           
 [65] "FREL"         "FRELLAST"     "FRELLASTi"    "FTAUi"        "KEL"          "KELC0"        "KELNOPT"      "KELR"        
 [73] "KELRSQ"       "KELRSQA"      "KELTMHI"      "KELTMLO"      "LASTTIME"     "LASTTIMEi"    "MRTEVIFO"     "MRTEVIFOi"   
 [81] "MRTEVIFP"     "MRTEVIFPi"    "MRTLAST"      "MRTLASTi"     "PTF"          "PTFi"         "PTR"          "PTRi"        
 [89] "PTROUGHRi"    "PTROUGHRENDi" "RACi"         "RACCMAXi"     "RACCMINi"     "RSSi"         "TAU"          "TAUi"        
 [97] "THALF"        "THALFF"       "TLAG"         "TLAST"        "TLASTi"       "TMAX"         "TMAXi"        "TMIN"        
[105] "TMINi"        "TOLD"         "TOLDi"        "VZFO"         "VZFOW"        "VZFP"         "VZFPW"        "VZFTAUi"     
[113] "VZFTAUWi"    
> 
> k <- parameter_required("OUTPUTUNIT$", names(mct), simplify=FALSE)
> mct[,names(k)]
  TIMEOUTPUTUNIT AMOUNTOUTPUTUNIT DOSEOUTPUTUNIT VOLUMEOUTPUTUNIT CONCOUTPUTUNIT KELOUTPUTUNIT CLOUTPUTUNIT AUCOUTPUTUNIT
1             HR               NA             MG                L          NG/ML          1/HR         L/HR      NG.HR/ML
  AUMCOUTPUTUNIT AUCNORMOUTPUTUNIT AUROUTPUTUNIT CONCNORMOUTPUTUNIT RATEOUTPUTUNIT VOLUMENORMOUTPUTUNIT CLNORMOUTPUTUNIT
1    NG.HR.HR/ML       NG.HR/ML/MG            NA           NG/ML/MG             NA                 L/KG         ML/HR/KG
> 
> k <- parameter_required("^AUC.[0-9]+?.T[1-2]$", names(mct), simplify=FALSE)
> mct[,names(k)]
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2
1        0        7        8       48        0       64        0      168
> 
> table(d$PKPTMS, d$TREATXT)
     
      1 mg 10 mg 100 mg 3 mg 30 mg 300 mg 600 mg 900 mg
  0      6     6      6    6     6      6      6      6
  0.5    6     6      6    6     6      6      6      6
  1      6     6      6    6     6      6      6      6
  1.5    6     6      0    6     0      0      0      0
  2      6     6      6    6     6      6      6      6
  2.5    6     6      0    6     0      0      0      0
  4      6     6      6    6     6      6      6      6
  8      6     6      6    6     6      6      6      6
  12     6     6      6    6     6      6      6      6
  16     6     6      6    6     6      6      6      6
  24     6     6      6    6     6      6      6      6
  48     6     6      6    6     6      6      6      6
  72     6     6      6    6     6      6      6      6
  96     0     0      6    0     6      6      6      6
  120    0     0      6    0     6      6      6      6
> 
> (tclabel <- paste0("results.", testcase.id, ".", mct$MODEL, ".", mct$DOSINGTYPE, ".", mct$AUCMETHOD))
[1] "results.tc010.M1.SD.LINLOG"
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : 
  Error in if (time[e] == t_last) {: missing value where TRUE/FALSE needed
For SDEID 2925168
In addition: Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in if (time[e] == t_last) {: missing value where TRUE/FALSE needed
For SDEID 2925168 
```

**Comment by jhhughes256** [comment]
The error now seen for tc010 M1SD is the same as tc012 M1SS (#27) when run on the past commit.

**Comment by tensfeldt** [comment]
#36 remains an issue with commit  96f6417

## tc010 with all concentations set to NA with several partial AUCs defined

```
> model_spec(mct)
[1] "openNCA: Model: M1 DosingType: SD AUCMETHOD: LINLOG"
> d$PKCNCN <- rep(NA, nrow(d)) # computation engine should not FALL OVER
> parameterset
   parameterset 
"PARAMETERLIST" 
> 
> # Parmeters for M1SD
> model_parameters("M1|M1SD")
  [1] "AUCALL"       "AUCALLDN"     "AUCDN"        "AUCINFO"      "AUCINFOC"     "AUCINFODN"    "AUCINFOi"     "AUCINFP"     
  [9] "AUCINFPC"     "AUCINFPDN"    "AUCINFPi"     "AUCLAST"      "AUCLASTi"     "AUCLASTC"     "AUCLASTCi"    "AUCLASTDN"   
 [17] "AUCLASTDNi"   "AUCT"         "AUCT1_T2"     "AUCTAUDNi"    "AUCTAU"       "AUCTAUi"      "AUCTDN"       "AUCXPCTO"    
 [25] "AUCXPCTOi"    "AUCXPCTP"     "AUCXPCTPi"    "AUMCINFO"     "AUMCINFOi"    "AUMCINFP"     "AUMCINFPi"    "AUMCLAST"    
 [33] "AUMCLASTi"    "AUMCTAUi"     "AUMCXPTO"     "AUMCXPTOi"    "AUMCXPTP"     "AUMCXPTPi"    "CAVi"         "CEST"        
 [41] "CLAST"        "CLASTi"       "CLFO"         "CLFOW"        "CLFP"         "CLFPW"        "CLFTAUi"      "CLFTAUWi"    
 [49] "CMAX"         "CMAXi"        "CMAXDN"       "CMAXDNi"      "CMIN"         "CMINi"        "CMINDN"       "CMINDNi"     
 [57] "CTROUGHi"     "CTROUGHENDi"  "DIi"          "DOSE"         "DOSEi"        "DOSEC"        "DOSECi"       "F"           
 [65] "FREL"         "FRELLAST"     "FRELLASTi"    "FTAUi"        "KEL"          "KELC0"        "KELNOPT"      "KELR"        
 [73] "KELRSQ"       "KELRSQA"      "KELTMHI"      "KELTMLO"      "LASTTIME"     "LASTTIMEi"    "MRTEVIFO"     "MRTEVIFOi"   
 [81] "MRTEVIFP"     "MRTEVIFPi"    "MRTLAST"      "MRTLASTi"     "PTF"          "PTFi"         "PTR"          "PTRi"        
 [89] "PTROUGHRi"    "PTROUGHRENDi" "RACi"         "RACCMAXi"     "RACCMINi"     "RSSi"         "TAU"          "TAUi"        
 [97] "THALF"        "THALFF"       "TLAG"         "TLAST"        "TLASTi"       "TMAX"         "TMAXi"        "TMIN"        
[105] "TMINi"        "TOLD"         "TOLDi"        "VZFO"         "VZFOW"        "VZFP"         "VZFPW"        "VZFTAUi"     
[113] "VZFTAUWi"    
> 
> k <- parameter_required("OUTPUTUNIT$", names(mct), simplify=FALSE)
> mct[,names(k)]
  TIMEOUTPUTUNIT AMOUNTOUTPUTUNIT DOSEOUTPUTUNIT VOLUMEOUTPUTUNIT CONCOUTPUTUNIT KELOUTPUTUNIT CLOUTPUTUNIT AUCOUTPUTUNIT
1             HR               NA             MG                L          NG/ML          1/HR         L/HR      NG.HR/ML
  AUMCOUTPUTUNIT AUCNORMOUTPUTUNIT AUROUTPUTUNIT CONCNORMOUTPUTUNIT RATEOUTPUTUNIT VOLUMENORMOUTPUTUNIT CLNORMOUTPUTUNIT
1    NG.HR.HR/ML       NG.HR/ML/MG            NA           NG/ML/MG             NA                 L/KG         ML/HR/KG
> 
> k <- parameter_required("^AUC.[0-9]+?.T[1-2]$", names(mct), simplify=FALSE)
> mct[,names(k)]
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2
1        0        7        8       48        0       64        0      168
> 
> table(d$PKPTMS, d$TREATXT)
     
      1 mg 10 mg 100 mg 3 mg 30 mg 300 mg 600 mg 900 mg
  0      6     6      6    6     6      6      6      6
  0.5    6     6      6    6     6      6      6      6
  1      6     6      6    6     6      6      6      6
  1.5    6     6      0    6     0      0      0      0
  2      6     6      6    6     6      6      6      6
  2.5    6     6      0    6     0      0      0      0
  4      6     6      6    6     6      6      6      6
  8      6     6      6    6     6      6      6      6
  12     6     6      6    6     6      6      6      6
  16     6     6      6    6     6      6      6      6
  24     6     6      6    6     6      6      6      6
  48     6     6      6    6     6      6      6      6
  72     6     6      6    6     6      6      6      6
  96     0     0      6    0     6      6      6      6
  120    0     0      6    0     6      6      6      6
> 
> (tclabel <- paste0("results.", testcase.id, ".", mct$MODEL, ".", mct$DOSINGTYPE, ".", mct$AUCMETHOD))
[1] "results.tc010.M1.SD.LINLOG"
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : 
  Error in if (time[e] == t_last) {: missing value where TRUE/FALSE needed
For SDEID 2925168
In addition: Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in if (time[e] == t_last) {: missing value where TRUE/FALSE needed
For SDEID 2925168 
```
## tc010 will not run with all concentrations set to missing, i.e. "NA"

```
> d$PKCNCN <- rep(NA, nrow(d)) # computation engine should not FALL OVER
> k <- parameter_required("^AUC.[0-9]+?.T[1-2]$", names(mct), simplify=FALSE)
> mct[,names(k)]
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2
1        0        7        8       48        0       64        0      168
> 
> mct <- mct[,-match(names(k), names(mct))]
> 
> k <- parameter_required("^AUC.[0-9]+?.T[1-2]$", names(mct), simplify=FALSE)
> mct[,names(k)]
data frame with 0 columns and 1 row
> 
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : 
  Error in if (time[e] == t_last) {: missing value where TRUE/FALSE needed
For SDEID 2925168
In addition: Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in if (time[e] == t_last) {: missing value where TRUE/FALSE needed
For SDEID 2925168 
```

**Comment by rudraya** [comment]
This issue was addressed in 6aba420, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
The 6aba420 commit resolves the issue with the crash and resulting parameter dataset being empty but introduces a new issue that is unexpected and not desirable.

The flags dataset returned by the computation engine only has 6 records. The original flags dataset had 624 records. It appears as though the processing of the output flags dataset was interrupted by not having any parameters generated. The output flags dataset MUST have the same # of records and provide the original content of the flags dataset if not parameter computations are performed.

## output Flags

```
  PKDATAROWID FLGEXSDE FLGEXKEL FLGEXAUC FLGEXST NDOSEI PKPCOM MIDPNTN MIDPNTA MIDPNTNU MIDPNTAU        RATEN        RATEA RATENU
1    66607600        0        1        0       0      1   <NA>      12 12.0665       HR       HR  530774.0155  527848.8531  NG/HR
2    66607588        0        1        0       0      1   <NA>      12 12.0415       HR       HR  508889.4894  507135.6453  NG/HR
3    66607596        0        1        0       0      1   <NA>      12 12.0415       HR       HR 1349531.6678 1344880.6223  NG/HR
4    66607590        0        1        0       0      1   <NA>      12 12.0415       HR       HR  705953.3538  703520.3459  NG/HR
5    66607594        0        1        0       0      1   <NA>      12 12.0415       HR       HR  682308.8235  679957.3045  NG/HR
6    66607602        0        1        0       0      1   <NA>      12 12.0415       HR       HR  408645.7353  407237.3727  NG/HR
  RATEAU
1  NG/HR
2  NG/HR
3  NG/HR
4  NG/HR
5  NG/HR
6  NG/HR
```


## Input flags
```
   PKDATAROWID FLGEXSDE FLGEXKEL FLGEXAUC FLGEXST NDOSEI PKPCOM MIDPNTN MIDPNTA MIDPNTNU MIDPNTAU RATEN RATEA RATENU RATEAU
1     22289744        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
2     22289745        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
3     22289746        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
4     22289747        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
5     22289748        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
6     22289749        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
7     22289750        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
8     22289751        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
9     22289752        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
10    22289753        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
11    22289754        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
12    22289755        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
13    22289756        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
14    22289562        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
15    22289563        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
16    22289564        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
17    22289565        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
18    22289566        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
19    22289567        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
20    22289568        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
21    22289569        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
22    22289570        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
23    22289571        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
24    22289572        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
25    22289573        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
26    22289574        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
27    22289523        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
28    22289524        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
29    22289525        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
30    22289526        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
31    22289527        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
32    22289528        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
33    22289529        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
34    22289530        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
35    22289531        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
36    22289532        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
37    22289533        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
38    22289534        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
39    22289535        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
40    22289588        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
41    22289589        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
42    22289590        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
43    22289591        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
44    22289592        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
45    22289593        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
46    22289594        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
47    22289595        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
48    22289596        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
49    22289597        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
50    22289598        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
51    22289599        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
52    22289600        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
53    22289757        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
54    22289758        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
55    22289759        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
56    22289760        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
57    22289761        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
58    22289762        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
59    22289763        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
60    22289764        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
61    22289765        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
62    22289766        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
63    22289767        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
64    22289768        0        0        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
65    22289769        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
66    22290446        0        1        0       0      1     NA      NA      NA       NA       NA    NA    NA     NA     NA
 [ reached 'max' / getOption("max.print") -- omitted 558 rows ]

```

**Comment by tensfeldt** [comment]
Issue #36 can be closed since this behavior is apparently resolved with commit 6aba420. However issue #27 is still occuring with tc013.
